spring-cloud-gcp-pubsub/src/test/java/com/google/cloud/spring/pubsub/core/subscriber/PubSubSubscriberTemplateTests.java

```diff
@@ -530,6 +533,17 @@ void testPullAndConvertAsync()
     assertThat(result.get(0).getProjectSubscriptionName().getSubscription()).isEqualTo("sub2");
   }
 
+  @Test
+  @Timeout(value = 5)
```

Comment:
Would you add a unit? Without a unit, I had to look up the API reference (https://junit.org/junit5/docs/5.5.1/api/org/junit/jupiter/api/Timeout.html).